source/math/FloatVectorList.ooc

```diff
@@ -24,12 +24,18 @@ FloatVectorList: class extends VectorList<Float> {
 		this super()
 	}
 	init: func ~heap (capacity: Int) {
-		super(capacity)
+		this super(capacity)
```

Comment:
Now that you mention it, neither am I. `init~default` had it, but none of the others. I should have removed them all instead. Will fix.